third_party/mbedtls/mbedtls-config.h

```diff
@@ -2583,7 +2683,11 @@ __inline int windows_kernel_snprintf(char * s, size_t n, const char * format, ..
 //#define MBEDTLS_SSL_CACHE_DEFAULT_MAX_ENTRIES      50 /**< Maximum entries in cache */
 
 /* SSL options */
+#if OPENTHREAD_ENABLE_APPLICATION_COAP_SECURE
+#define MBEDTLS_SSL_MAX_CONTENT_LEN             2*768 /**< Maxium fragment length in bytes, determines the size of each of the two internal I/O buffers */
```

Comment:
i can delete it. we have used payload, which was over 1k. but it's not necessary by default. removed -> done.

Reply:
sorry, it's not really true, what i say before. the certificate alone is 823 Bytes long. then we must add the coap header to can transfer all nessesary's. with 900 it will work. so i make the same define with 900. note: if a user add a certificate, which is greater than our's, the dtls handshake will fail.